android/src/main/java/io/parity/signer/ui/helpers/PreviewData.kt

```diff
@@ -0,0 +1,11 @@
+
+	val exampleQRCode: List<UByte>  = listOf<Int>(137, 80, 78, 71, 13, 10, 26, 10, 0, 0, 0, 13, 73, 72, 68, 82, 0, 0, 0, 196, 0, 0, 0, 196, 1, 3, 0, 0, 0, 250, 104, 28, 40, 0, 0, 0, 6, 80, 76, 84, 69, 0, 0, 0, 255, 255, 255, 165, 217, 159, 221, 0, 0, 2, 48, 73, 68, 65, 84, 120, 156, 237, 215, 205, 113, 237, 32, 12, 5, 96, 168, 0, 58, 112, 255, 101, 209, 129, 168, 192, 239, 147, 125, 243, 178, 201, 82, 204, 100, 17, 50, 215, 54, 58, 154, 209, 223, 65, 34, 253, 110, 63, 175, 253, 135, 252, 18, 164, 183, 54, 214, 140, 43, 230, 110, 45, 174, 117, 121, 141, 168, 71, 102, 52, 178, 152, 49, 246, 104, 123, 190, 130, 19, 8, 65, 164, 100, 183, 171, 177, 127, 191, 130, 35, 200, 30, 49, 87, 187, 118, 204, 214, 78, 34, 193, 188, 240, 218, 188, 211, 141, 20, 28, 64, 8, 218, 246, 184, 251, 136, 126, 243, 193, 247, 1, 36, 25, 130, 122, 223, 139, 51, 47, 67, 106, 145, 103, 205, 52, 187, 230, 8, 223, 207, 170, 71, 122, 67, 193, 164, 162, 160, 87, 166, 212, 82, 203, 122, 100, 72, 231, 224, 195, 154, 77, 82, 149, 141, 214, 19, 169, 77, 33, 50, 215, 245, 124, 62, 204, 192, 119, 193, 146, 214, 35, 194, 28, 83, 116, 177, 145, 68, 102, 251, 77, 227, 0, 130, 124, 236, 138, 19, 15, 213, 111, 116, 53, 164, 127, 0, 225, 131, 190, 183, 240, 124, 204, 157, 39, 88, 27, 172, 71, 216, 103, 89, 103, 109, 11, 144, 154, 132, 143, 7, 118, 133, 136, 252, 237, 225, 77, 37, 174, 204, 172, 223, 9, 164, 231, 188, 240, 39, 161, 116, 208, 63, 218, 21, 7, 16, 249, 116, 176, 166, 238, 96, 131, 135, 147, 19, 136, 105, 67, 94, 136, 128, 152, 206, 99, 101, 223, 180, 115, 113, 210, 47, 71, 236, 85, 44, 158, 40, 157, 178, 232, 236, 127, 24, 82, 138, 244, 48, 145, 146, 137, 182, 140, 231, 243, 139, 241, 165, 72, 150, 142, 221, 91, 219, 203, 150, 231, 132, 193, 31, 15, 106, 17, 6, 85, 77, 94, 169, 36, 233, 253, 64, 245, 136, 80, 133, 54, 201, 204, 116, 166, 245, 138, 172, 97, 57, 210, 25, 220, 99, 177, 234, 12, 107, 22, 62, 72, 235, 145, 249, 114, 2, 168, 96, 95, 187, 199, 131, 207, 119, 17, 98, 35, 204, 102, 98, 40, 160, 11, 138, 184, 131, 79, 229, 72, 150, 42, 57, 255, 148, 45, 12, 94, 252, 160, 82, 143, 152, 23, 110, 116, 146, 153, 124, 236, 20, 20, 242, 245, 160, 20, 113, 115, 200, 46, 78, 252, 166, 82, 175, 160, 90, 143, 232, 18, 230, 171, 137, 1, 200, 139, 49, 129, 199, 9, 228, 214, 33, 146, 230, 14, 239, 242, 54, 14, 15, 33, 170, 21, 218, 44, 166, 56, 90, 54, 95, 55, 226, 82, 36, 87, 26, 151, 212, 149, 23, 47, 243, 66, 212, 245, 200, 99, 81, 95, 50, 251, 132, 44, 175, 52, 53, 168, 114, 4, 203, 223, 146, 189, 98, 214, 9, 79, 32, 206, 148, 255, 140, 250, 59, 1, 167, 169, 203, 149, 67, 136, 229, 236, 178, 143, 34, 217, 54, 232, 28, 65, 120, 144, 252, 35, 48, 216, 245, 166, 79, 70, 107, 17, 18, 30, 232, 176, 38, 186, 226, 105, 27, 65, 183, 28, 249, 207, 16, 215, 98, 217, 12, 188, 252, 120, 80, 139, 252, 188, 254, 144, 95, 129, 252, 3, 117, 24, 116, 223, 174, 44, 31, 82, 0, 0, 0, 0, 73, 69, 78, 68, 174, 66, 96, 130).map {it.toUByte()}
+}
```

Comment:
Correct me if I am wrong but these can be just plain images in `R.drawable` or something?

Reply:
This is used for preview (development) only. 
In our data model we receiving from rust array of bytes to construct bitmap. This is not a usual way to pass images, while probably best available for FFI. 
So for this case I think it's better to have mocked data model and check image is properly constructed rather than separate logic to show similar drawable resource in preview rather than using data model actually used in the app.